exams/2006/pagrindinis/u2/Duom2.txt

```diff
@@ -0,0 +1,7 @@
+6 70 10 15
+Elektrėnai      50.5
+Žiežmariai      20
+Kaunas          22.35
+Raseiniai       80
+Kryžkalnis      20
+Klaipėda        100.8
```

Comment:
There is no need for this file, because this file is created dynamically via `tests.py` file into a temporary directory.